From d723c9e7e3da2f247dce8225ffa67da904d1dbb0 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Wed, 1 Dec 2021 15:43:34 +0800
Subject: [PATCH] no warn if we find content in the deployment's /var

Since commit [edf7477e deploy: Warn if we find content in the
deployment's /var] applied, warns in LAT but harmless

Revert the commit for no warn

Upstream-Status: Inappropriate [LAT specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 src/libostree/ostree-sysroot-deploy.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/libostree/ostree-sysroot-deploy.c b/src/libostree/ostree-sysroot-deploy.c
index 039f5f50..fe439f56 100644
--- a/src/libostree/ostree-sysroot-deploy.c
+++ b/src/libostree/ostree-sysroot-deploy.c
@@ -2784,9 +2784,6 @@ sysroot_initialize_deployment (OstreeSysroot     *self,
                                cancellable, error))
     return FALSE;
 
-  if (!lint_deployment_fs (self, new_deployment, deployment_dfd, cancellable, error))
-    return FALSE;
-
   ot_transfer_out_value (out_new_deployment, &new_deployment);
   return TRUE;
 }
-- 
2.27.0

